packages/session/lib/session.ts

```diff
@@ -1,9 +1,45 @@
-const cookieSignature = require('cookie-signature');
-const expressSession = require('express-session');
-const cookie = require('cookie');
+// @ts-ignore
+import cookieSignature from 'cookie-signature';
+import expressSession from 'express-session';
+import type { SessionOptions } from 'express-session';
+import { Request, Response, NextFunction } from 'express';
+import cookie from 'cookie';
+
+// FIXME: In the future this types will need to come from Keystone itself.
+type _Item = { id: string };
+type _List = {
+  key: string;
+  adapter: { itemsQuery: (args: { where: { id: string } }) => Promise<_Item[]> };
+};
+type _Keystone = { lists: Record<string, _List> };
+
+declare module 'express' {
```

Comment:
This module augmentation(and the one below it) will probably cause issues. This means that anyone who imports @keystonejs/session(including transitively) will see these definitions and it could be wrong/conflict with the users types, you should probably remove it.